app/controllers/account/profile.js

```diff
@@ -1,4 +1,20 @@
 import Controller from '@ember/controller';
 
 export default Controller.extend({
+  actions: {
+    updateContactInfo() {
+      this.set('isLoading', true);
+      let currentUser = this.model;
+      currentUser.save()
```

Comment:
IMO async-await would make this less verbose

Reply:
@niranjan94 I agree. Made it easier like that.